src/utils.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports =
+  strContains: (str, searchValue) ->
+    str && str.toLowerCase().indexOf(searchValue) > -1
```

Comment:
The version used here does not seems to depend on lodash, I'm gonna use lodash standalone packs